owner-onboarding-server/src/main.rs

```diff
@@ -346,6 +415,48 @@ async fn main() -> Result<()> {
             .context("Error converting owner public key to PK")?
     };
 
+    // Voucher registration times
+    let str_registration_period = settings.registration_period;
+    let str_re_registration_window = settings.re_registration_window;
+    let registration_period = match str_registration_period {
+        Some(value) => {
+            let tmp = value
+                .parse::<u32>()
+                .context("must provide a valid registration_period")?;
+            if tmp == 0 {
+                bail!("registration_period cannot be 0");
+            }
+            tmp
+        }
+        None => DEFAULT_REGISTRATION_PERIOD,
+    };
+    let re_registration_window = match str_re_registration_window {
+        Some(value) => {
+            let tmp = value
+                .parse::<u32>()
+                .context("must provide a valid re_registration_window")?;
+            if tmp == 0 {
+                bail!("re_registration_window cannot be 0");
+            }
+            if tmp >= registration_period {
```

Comment:
This check seems to be duplicated, as it's also executed later on.

Reply:
Yes, since the `registration_window` and `re_registration_window` are not compulsory fields, a user may choose to config one, both or neither of them. In the case that only one of them is configured the other one will get the default value, thereby we need to check again that  window < registration_period.

Reply:
Yep, I was initially going to type it would be best to move this down, and then I saw it was also there. So maybe just only use that one, and drop this one?

Reply:
Will do.